db/db_impl.h

```diff
@@ -1424,6 +1425,8 @@ class DBImpl : public DB {
 
   // Flag to check whether Close() has been called on this DB
   bool closed_;
+
+  std::unique_ptr<ErrorHandler> error_handler_;
```

Comment:
Why a pointer rather than `ErrorHandler error_handler_`?

Reply:
I wanted to explicitly construct it passing DBImpl::mutex_ and DBImpl::immutable_db_options_. Although it occurs to me now that I can call the constructor during DBImpl construction.